sources/api/apiclient/src/main.rs

```diff
@@ -462,13 +462,12 @@ async fn run() -> Result<()> {
     let (args, subcommand) = parse_args(env::args());
     trace!("Parsed args for subcommand {:?}: {:?}", subcommand, args);
 
-    // We use TerminalMode::Stderr because apiclient users expect server response data on stdout.
-    TermLogger::init(
+    // We use SimpleLogger because apiclient users expect server response data on stdout.
```

Comment:
Actually, reviewing this change made me realize that the old comment was even more important than I thought, and that we shouldn't use SimpleLogger here.  We weren't using `Mixed` mode like in the other files, we were using straight `Stderr` mode, because we want to totally control what goes to stdout.  Only response data from the server should be on stdout, no logging information whatsoever, which should all go to stderr.

So, I think we need to revert the changes in this file.  Sorry I missed this earlier!

Reply:
Ah missed the notification. Will update